### DISCUSSION TITLE
Netlify CLI Freemium Forever - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#23] [Comments]
# Netlify CLI Freemium Forever - Tulisan Programmer Gadungan

Bismillah .... Sebetulnya posting kali ini lebih ke arah opini pribadi aja, soalnya project yang running di netlify udah kadung banyak, jadi…

https://topidesta.my.id/netlify-cli-freemium-forever/

**Comment by mdestafadilah** [comment]
baiklah

**Comment by mdestafadilah** [comment]
https://login.rsuppersahabatan.co.id/

ini pake netlify-cli buat build ke production dari git organisasi private
